associateLambda/index.ts

```diff
@@ -0,0 +1,43 @@
+import * as indexModule from './index'
+
+// lambda handler function
+
+export interface AssocEvent {
```

Comment:
If the methods are named as `xxxxxAssociate`, and is imported as `associateLambda`, for consistency please refactor this interface to `AssociateEvent`.